_commands/toys/linuxlogo.md

```diff
@@ -0,0 +1,39 @@
+---
+---
+
+linuxlogo
+-------
+`linuxlogo`  is a Linux command line utility that generates a color ANSI picture of logo with a few system information.
+
+To use `linuxlogo`, simply type:
+
+~~~ bash
+$ sudo apt install linuxlogo
+$ linuxlogo 
+~~~
+
+<!--more-->
+
+### Useful Commands / Examples
+- `-a` generate the linux logo without color
+- `-u` also display system uptime
+- `-l` print LOGO only and exclude all other System Information
+- `-L list` see other built-in logos for various Linux distributions
+- `-L NUM` print logo with number NUM
+- `-L NAME` print logo with name NAME
+
+#### Example command
+
+~~~bash
+$ linuxlogo 
+~~~
+![linuxlogo](https://www.tecmint.com/wp-content/uploads/2015/06/Get-Default-OS-Logo.png)
+
+~~~bash
+$ linuxlogo -L 1

```

Comment:
Adding a flag with just a screenshot isn't very helpful documentation, could you verbally describe what is happening as well? What does the -L flag add?